### PR TITLE
Issue 9544

### PR DESCRIPTION
**Pull request by schlameel**
### Describe your changes :

Issue #9544
I worked on the ingestion API - patch_mixin - to add the ability to modify the owner of an entity.

#
### Type of change :

- [ ] Bug fix
- [ ] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
N/A

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Comment by TeddyCr** [comment]
Thanks @schlameel for this PR. Would you be able to run `make py_format` and then `make py_format_check` to check if there are any linting issues? https://github.com/open-metadata/OpenMetadata/pull/10310#issuecomment-1442761423

File: ingestion/src/metadata/ingestion/ometa/mixins/patch_mixin.py [hunk 1]
```diff
@@ -15,21 +15,28 @@
...
 from metadata.ingestion.ometa.client import REST
 from metadata.ingestion.ometa.utils import model_str
 from metadata.utils.helpers import find_column_in_table_with_index
 from metadata.utils.logger import ometa_logger
 
+try:
+    from typing import get_args
+except ImportError:
+    from typing_compat import get_args
+
```
Comment (ulixius9) [inline ingestion/src/metadata/ingestion/ometa/mixins/patch_mixin.py:35]: @schlameel are we using this imports anywhere?

File: ingestion/src/metadata/ingestion/ometa/mixins/patch_mixin.py [hunk 2]
```diff
@@ -15,21 +15,28 @@
...
 logger = ometa_logger()
 
 T = TypeVar("T", bound=BaseModel)
+U = TypeVar("U", bound=BaseModel)
```
Comment (ulixius9) [inline ingestion/src/metadata/ingestion/ometa/mixins/patch_mixin.py:39]: also, this one, are we using this TypeVar anywhere? if not then can we remove this?

**Review by ulixius9 — COMMENTED** [review]
@schlameel Thanks for the PR some quick comments

File: ingestion/src/metadata/ingestion/ometa/mixins/patch_mixin.py (hunk 2 above)
Comment (schlameel) [reply ingestion/src/metadata/ingestion/ometa/mixins/patch_mixin.py:39]: @ulixius9 Good catch.  I changed the parameters in the method I added and no longer needed these.  I cannot figure out how to update the code and push to the same branch, so I'm going to close this PR and open a new one with the requested changes.

**Comment by schlameel** [comment]
Making changes requested by @ulixius9 and creating a new PR.

**Pull request closed without merging**